doc/dataset_list.rst

```diff
@@ -0,0 +1,198 @@
+        <td>HCP842 bundle atlas</td>
+        <td>Human</td>
+        <td>Whole brain/bundle-wise tractograms in MNI space; 80 bundles</td>
+        <td>Human Connectome Project (HCP) scanner</td>
+        <td>bundle_atlas_hcp842</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>HCP bundle FA</td>
+        <td>Human</td>
+        <td>Fractional Anisotropy (FA); 2 bundles</td>
+        <td></td>
+        <td>bundle_fa_hcp</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>HCP tractogram</td>
+        <td>Human/td>
+        <td>Whole brain tractogram</td>
+        <td>Human Connectome Project (HCP) scanner</td>
+        <td>target_tractogram_hcp</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>ISBI 2013</td>
+        <td>Phantom</td>
+        <td>Multi shell data: b-vals: [0, 1500, 2500] (s/mm^2); 64 gradient directions</td>
+        <td></td>
+        <td>isbi2013_2shell</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>IVIM dataset</td>
+        <td>Human</td>
+        <td>Multi shell data: b-vals: [0, 10, 20, 30, 40, 60, 80, 100, 120, 140, 160, 180, 200, 300, 400, 500, 600, 700, 800, 900, 1000] (s/mm^2); 21 gradient directions</td>
+        <td></td>
+        <td>fetch_ivim</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>MNI template</td>
+        <td>Human</td>
+        <td>MNI 2009a T1, T2; 2009c T1, T1 mask</td>
+        <td></td>
+        <td>mni_template</td>
+        <td><a href='http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin2009'>ICBM 152 Nonlinear atlases version 2009</a>. VS Fonov, AC Evans, K Botteron, CR Almli, RC McKinstry, DL Collins and BDCG, Unbiased average age-appropriate atlases for pediatric studies, NeuroImage,Volume 54, Issue 1, January 2011, ISSN 1053–8119, doi:10.1016/j.neuroimage.2010.07.033; VS Fonov, AC Evans, RC McKinstry, CR Almli and DL Collins, Unbiased nonlinear average age-appropriate brain templates from birth to adulthood, NeuroImage, Volume 47, Supplement 1, July 2009, Page S102 Organization for Human Brain Mapping 2009 Annual Meeting, doi:10.1016/S1053-8119(09)70884-5</td>
+    </tr>
+    <tr>
+        <td>qt-dMRI C57Bl6 mice dataset</td>
+        <td>Animal</td>
+        <td>2 C57Bl6 mice test-retest qt-dMRI; Corpus callosum (CC) bundle masks</td>
+        <td></td>
+        <td>qtdMRI_test_retest_2subject</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>SCIL b0</td>
+        <td></td>
+        <td>b0</td>
+        <td>GE (1.5, 3 T), Philips (3 T); Siemens (1.5, 3 T)</td>
+        <td>scil_b0</td>
+        <td><a href='http://scil.dinf.usherbrooke.ca/fr/'>Sherbrooke Connectivity Imaging Lab (SCIL)</a></td>
+    </tr>
+    <tr>
+        <td>Sherbrooke 3 shells</td>
+        <td>Human</td>
+        <td>Multi shell data: b-vals: [0, 1000, 2000; 3500] (s/mm^2); 193 gradient directions</td>
+        <td></td>
+        <td>sherbrooke_3shell</td>
+        <td><a href='http://scil.dinf.usherbrooke.ca/fr/'>Sherbrooke Connectivity Imaging Lab (SCIL)</a></td>
```

Comment:
I think it is better to remove `/fr/`. I suppose the redirection is done following your browser language.

Reply:
Done in 873ad5e. Thanks.